lib/util/add.js

```diff
@@ -6,6 +6,24 @@ module.exports = add
 
 var own = {}.hasOwnProperty
 
+var NO_RULES = []
+
+// Add `rules` for `word` to the table.
+function addRules(dict, word, rules) {
+  // Some dictionaries will list the same word multiple times with different
+  // rule sets.
+  if (word in dict) {
+    var curr = dict[word]
+    if (curr === NO_RULES) {
+      dict[word] = rules.slice(0)
```

Comment:
is `.slice(0)` really faster than `concat()`? I kinda like the latter style and I’m not sure why the former would be faster?

Reply:
I think it was marginally faster, but probably the range of variation in the benchmark is greater than so I may have misinterpreted the result, I can't remember now what difference that made, I can try both again 👍

Reply:
I’m assuming V8 can deal with both as fast, or maybe a marginal difference indeed. In which case I’d prefer concat

Reply:
The first 3 runs are with `concat`, the next 3 are with `slice(0)`, the remaining 3 are with `concat` 🤷‍♂️ 

![image](https://user-images.githubusercontent.com/1812093/93782704-c7d60100-fc22-11ea-9c52-609624c37cdc.png)

Let's go with concat.